ui/components/multichain/app-header/app-header.js

```diff
@@ -158,14 +158,22 @@ export const AppHeader = ({ onClick }) => {
                 onClick={networkOpenCallback}
                 display={[DISPLAY.FLEX, DISPLAY.NONE]} // show on popover hide on desktop
               />
-              <PickerNetwork
-                margin={2}
-                label={currentNetwork?.nickname}
-                src={currentNetwork?.rpcPrefs?.imageUrl}
-                onClick={networkOpenCallback}
-                display={[DISPLAY.NONE, DISPLAY.FLEX]} // show on desktop hide on popover
-                className="multichain-app-header__contents__network-picker"
-              />
+              {popupStatus ? null : (
+                <Box
+                  display={DISPLAY.FLEX}
+                  className="multichain-app-header__contents__network-container"
+                >
```

Comment:
Instead of using display flex just use a div and the grid wrapper should handle the width so I think you can just do

```suggestion
                <div>
```